File: trunk/server/net/Protocols/RoundResponse.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// roundOver 协议:-25
    /// </summary>
	[Proto(value=-25,description="roundOver")]
	[ProtoContract]
	public class RoundResponse
	{
        /// <summary>
        ///  房间id
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public string roomId;
        /// <summary>
        ///  回合信息
        /// </summary>
		[ProtoMember(4, IsRequired = false)]
		public RoundInfo nri;
        /// <summary>
        ///  是否成功
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public bool success;
        /// <summary>
        ///  错误消息
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string info;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/RoundResponse.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.BaseResponse;
import org.ngame.protocol.annotation.Proto;
import org.ngame.protocol.domain.RoundInfo;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = -25, description = "roundOver")
public class RoundResponse extends BaseResponse
{
   @Protobuf(description = "房间id")
   private String roomId;
   @Protobuf(description = "回合信息")
   private RoundInfo nri;
}
